include/log4cplus/config.hxx

```diff
@@ -31,7 +31,11 @@
 #else
 #  include <log4cplus/config/defines.hxx>
 #endif
+#ifdef __cplusplus
 #include <cstddef>
+#else // __cplusplus
+#include <stddef.h>
+#endif
```

Comment:
Can you try without the `#else` branch? I think the `<cstddef>` is there only for the C++ definitions.